Spanner/src/Session/CacheSessionPool.php

```diff
@@ -349,11 +350,13 @@ public function release(Session $session)
             $name = $session->name();
 
             if (isset($data['inUse'][$name])) {
+                $creationTime = $data['inUse'][$name]['creation'];
```

Comment:
Are we positive this will always exist? What about for users who update? 

(this is why using arrays instead of types can be dangerous)

Reply:
`this will always exist` -> do you mean `['creation]` arg in the array? 
Yes!